docs/cloud-integration.md

```diff
@@ -245,17 +260,23 @@ mydataframe.write.format("parquet").save("s3a://bucket/destination")
 
 More details on these committers can be found in the latest Hadoop documentation.
 
+Note: depending upon the committer used, in-progress statistics may be
+under-reported with Hadoop versions before 3.3.1.
+Are
 ## Further Reading
 
 Here is the documentation on the standard connectors both from Apache and the cloud providers.
 
-* [OpenStack Swift](https://hadoop.apache.org/docs/current/hadoop-openstack/index.html).
+* [OpenStack Swift](https://hadoop.apache.org/docs/current/hadoop-openstack/index.html). Are
```

Comment:
extra Are

Reply:
sorry; my fault for having dictation turned on and not paying attention to what the music in the background was saying...